ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,211 @@
+
+        # For telescope images
+        if self.skip_images is False:
+            for group_image in group_path_list_images:
+                if group_image not in self.output_file:
+                    head, tail = os.path.split(group_image)
+                    self.output_file.create_group(head, tail, createparents=True)
+                for table in file.root[group_image]:
+                    if (group_image + '/' + table.name) in self.output_file:
+                        output_node = self.output_file.get_node(group_image
+                                                                + '/' + table.name)
+                        output_node.append(file.root[group_image + '/' + table.name][:])
+                    else:
+                        target_group = self.output_file.root[group_image]
+                        file.copy_node(table, newparent=target_group)
+
+        # For telescope parameters
+        if self.skip_parameters is False:
+            for group_parameter in group_path_list_parameters:
+                if group_parameter not in self.output_file:
+                    head, tail = os.path.split(group_parameter)
+                    self.output_file.create_group(head, tail, createparents=True)
+                for table in file.root[group_parameter]:
+                    if (group_parameter + '/' + table.name) in self.output_file:
+                        output_node = self.output_file.get_node(group_parameter
+                                                                + '/' + table.name)
+                        output_node.append(file.root[group_parameter + '/' + table.name][:])
+                    else:
+                        target_group = self.output_file.root[group_parameter]
+                        file.copy_node(table, newparent=target_group)
+
+    def start(self):
+        merged_files_counter = 0
+        len_input_path_list = len(self.input_path_list)
+
+        for i, current_file in tqdm(
+            enumerate(self.input_path_list),
+            desc="Merging",
+            total=(len_input_path_list),
+            unit="Files",
+            disable=not self.progress_bar
+        ):
+            # Check if Subarray from current file matches to first one
+            current_subarray = SubarrayDescription.from_hdf(current_file)
+            if self.first_subarray != current_subarray:
+                self.log.warning(f'Subarray does not match for {current_file}. Skip File')
+                continue
+
+            current_file_opened = tables.open_file(current_file, mode='r')
+            self.add_image_statistics(current_file_opened)
+            self.merge_tables(current_file_opened)
+            current_file_opened.close()
+
+            merged_files_counter += 1
+
+        self.log.info(f"{merged_files_counter} out of {len_input_path_list} Files "
+                       "had been merged!")
+
+    def finish(self):
+        pass
+
+
+def main():
+    tool = DL1MergeTool()
```

Comment:
I just learned that `Tools` have the `launch_instance` method.

So you can do: `DL1MergeTool.launche_instance()`

Reply:
have you tried it? I just get an error about two instances. Not sure how it is supposed to work

Reply:
Did you try this in `ipython`? Because I think it conflicts because ipython itself is already an instance of `Application` and only one can live per process.

Reply:
This works:

```
❯ python
Python 3.7.7 (default, Mar 26 2020, 15:48:22) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from ctapipe.core import Tool
>>> Tool.launch_instance()
INFO [Tool] (tool/initialize): ctapipe version 0.9.0.post9+gite02948a
```

Reply:
If i switch it to
```
def main():
    MergeTool.launch_instance()
```
it initializes and starts correctly, but doesn't do anything. Am i missing something?